receipt_chroma/receipt_chroma/exceptions.py

```
@@ -0,0 +1,60 @@
+        super().__init__(message)
+
+
+__all__ = [
+    "ReceiptChromaError",
+    "ChromaConfigurationError",
+    "ChromaClientStateError",
+    "ChromaClientClosedError",
+    "ChromaReadOnlyError",
+    "ChromaCollectionNotFoundError",
+    "ChromaDeltaUploadError",
+]
```

Ruff (0.16.0)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)